<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamerEditDialog</class>
 <widget class="QDialog" name="StreamerEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>238</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>12</number>
     </property>
     <property name="verticalSpacing">
      <number>12</number>
     </property>
     <property name="leftMargin">
      <number>12</number>
     </property>
     <property name="topMargin">
      <number>12</number>
     </property>
     <property name="rightMargin">
      <number>12</number>
     </property>
     <property name="bottomMargin">
      <number>12</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="buddy">
        <cstring>nameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="sourceUrlLabel">
       <property name="text">
        <string>Source URL:</string>
       </property>
       <property name="buddy">
        <cstring>sourceUrlEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="sourceUrlEdit">
       <property name="placeholderText">
        <string>rtsp://example.com/video</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="targetUrlLabel">
       <property name="text">
        <string>Target URL:</string>
       </property>
       <property name="buddy">
        <cstring>targetUrlEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="targetUrlEdit">
       <property name="placeholderText">
        <string>rtmp://rtmp.example.com/live</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="keyLabel">
       <property name="text">
        <string>Key:</string>
       </property>
       <property name="buddy">
        <cstring>keyEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="keyEdit">
       <property name="placeholderText">
        <string>xxxx-xxxx-xxxx-xxxx-xxxx</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="enabledLabel">
       <property name="text">
        <string>Enabled:</string>
       </property>
       <property name="buddy">
        <cstring>enabledCheckBox</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="enabledCheckBox">
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>sourceUrlEdit</tabstop>
  <tabstop>targetUrlEdit</tabstop>
  <tabstop>keyEdit</tabstop>
  <tabstop>enabledCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StreamerEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StreamerEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
